core-rust/state-manager/src/jni/test_state_reader.rs

```diff
@@ -112,7 +112,7 @@ pub enum TransactionOutcomeJava {
 #[derive(Debug, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
 pub struct JavaValidatorInfo {
     pub stake_unit_resource: ResourceAddress,
-    pub unstake_receipt_resource: ResourceAddress,
+    pub claim_receipt_resource: ResourceAddress,
```

Comment:
not sure why we have `receipt` in the name here?

Reply:
actually needs to through a PR https://github.com/radixdlt/babylon-node/pull/616